source/common/upstream/load_balancer_impl.cc

```diff
@@ -562,6 +562,30 @@ HostConstSharedPtr LoadBalancerContextBase::selectOverrideHost(const HostMap* ho
   return nullptr;
 }
 
+LoadBalancerContext::OverrideHostStatus LoadBalancerContextBase::createOverrideHostStatus(
```

Comment:
Is this used?

Reply:
Right, it is unnecessary after the refactoring.